.rubocop.yml

```diff
@@ -1589,6 +1603,10 @@ Style/NumericPredicate:
 Style/OneLineConditional:
   Enabled: true
 
+# TODO: refactor the use of OpenStruct out of the unit tests and enable
```

Comment:
Could this TODO use a label like `TEST REFACTOR`? Or `MINITEST UPGRADE`?

Reply:
I went with `UNIT TESTS` in 8ad6c42

Reply:
@fallwith - I think this change didn't make it into the commit? Same for the one on [test/multiverse/suites/rails/Envfile](https://github.com/newrelic/newrelic-ruby-agent/pull/1220/files/c082352a0cc2179ac84507da2532d7dd349e8b2c#diff-99bf1186c42234ea69c63a34fb5278ecd62d209485f62a0fc83eea80131f8a6d)

Reply:
Thank you! The change went to the wrong branch (`liblarr` instead of `murphy`). [8a4aaba](https://github.com/newrelic/newrelic-ruby-agent/pull/1220/commits/8a4aabadda7e4ad1bec2a635b3175ba228c50c7d) has now brought in the desired updates.